AsyncDisplayKit/ASCellNode+Internal.h

```diff
@@ -58,10 +60,7 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, strong, nullable) UICollectionViewLayoutAttributes *layoutAttributes;
 
-/// readwrite variant of the readonly public property.
-@property (nonatomic, copy, nullable) NSString *supplementaryElementKind;
-
-@property (nonatomic, copy, nullable) NSIndexPath *cachedIndexPath;
+@property (weak, nullable) ASCollectionElement *collectionElement;
```

Comment:
Does the collectionElement hold a strong reference to its cellNode? Shouldn't be the other way around?